### PR TITLE
ppc64le: CI/CD test scripts 

### PR DESCRIPTION
**Pull request by wdirons**
This PR enables unit test to be run on ppc64le by invoking ci_build.sh. For example:
./tensorflow/tools/ci_build/ci_build.sh gpu \
    --docker file tensorflow/tools/ci_build/Dockerfile.gpu.ppc64le \
    ./tensorflow/tools/ci_build/linux/ppc64le/gpu/run_py3_core.sh

This PR also fixes two other problems found while testing the scripts:
1) Since configure.py defaults to NCCL2 and NCCL is not available for ppc64le and CUDA 9.0, in the docker file we set TF_NCCL_VERSION to 1.

2) Test on python3 were seg faulting when using h5py, I modified the ppc64le specific script that installs h5py to install pip and then install h5py using pip instead of easy_install.

### CODE REVIEW

File: tensorflow/tools/ci_build/Dockerfile.gpu.ppc64le [hunk 1]
```diff
@@ -26,3 +26,6 @@ ENV LD_LIBRARY_PATH /usr/local/cuda/extras/CUPTI/lib64:$LD_LIBRARY_PATH
 # Configure the build for our CUDA configuration.
 ENV TF_NEED_CUDA 1
 ENV TF_CUDA_COMPUTE_CAPABILITIES 3.0
+
+#TODO get NCCL 2 in the docker image
```
Comment (gunan) [inline tensorflow/tools/ci_build/Dockerfile.gpu.ppc64le]: Nit, One space after # and before TODO

File: tensorflow/tools/ci_build/install/install_hdf5_ppc64le.sh [hunk 2]
```diff
@@ -24,7 +24,10 @@ apt-get install -y libhdf5-dev
 ln -s /usr/lib/powerpc64le-linux-gnu/libhdf5_serial.so /usr/lib/powerpc64le-linux-gnu/libhdf5.so
 ln -s /usr/lib/powerpc64le-linux-gnu/libhdf5_serial_hl.so /usr/lib/powerpc64le-linux-gnu/libhdf5_hl.so
 
-#pip is not installed yet, so use easy_install
+#Install pip here because h5py seg faults with python3 when installed via easy_install
```
Comment (gunan) [inline tensorflow/tools/ci_build/install/install_hdf5_ppc64le.sh]: Is there a github issue, or bug we can link to here?

File: tensorflow/tools/ci_build/linux/ppc64le/cpu/run_cc_core.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```
Comment (gunan) [inline tensorflow/tools/ci_build/linux/ppc64le/cpu/run_cc_core.sh]: For our scripts, I split them up to speed up the presubmits.
However you may want to merge them, as separate scripts will need to redundantly build everything.
So you may simply have two scripts, "run_py2.sh" and "run_py3.sh" and they can simply build and test "//tensorflow/... -//tensorflow/compiler/..."

File: tensorflow/tools/ci_build/install/install_hdf5_ppc64le.sh (hunk 2 above)
Comment (wdirons) [reply tensorflow/tools/ci_build/install/install_hdf5_ppc64le.sh]: There is no github issue , I will create one.

File: tensorflow/tools/ci_build/linux/ppc64le/cpu/run_cc_core.sh (hunk 3 above)
Comment (wdirons) [reply tensorflow/tools/ci_build/linux/ppc64le/cpu/run_cc_core.sh]: Thanks for the suggestion, I will update the code to run this way.

File: tensorflow/tools/ci_build/install/install_hdf5_ppc64le.sh (hunk 2 above)
Comment (wdirons) [reply tensorflow/tools/ci_build/install/install_hdf5_ppc64le.sh]: I tried today to recreate this issue and was unable to, I'm not sure what was going on. I will remove this change.

**Comment by wdirons** [comment]
@gunan, I added the fixes this pull request. Thank you.

**Review by gunan — APPROVED** [review]

**Pull request merged**